swap/swap.go

```diff
@@ -199,15 +204,28 @@ func New(dbPath string, prvkey *ecdsa.PrivateKey, backendURL string, params *Par
 		factory,
 	)
 	// start the chequebook
-	if swap.contract, err = swap.StartChequebook(chequebookAddressFlag, initialDepositAmountFlag); err != nil {
-		return nil, err
-	}
-	availableBalance, err := swap.AvailableBalance()
-	if err != nil {
+	if swap.contract, err = swap.StartChequebook(chequebookAddressFlag); err != nil {
 		return nil, err
 	}
 
-	swapLog.Info("available balance", "balance", availableBalance)
+	// deposit money in the chequebook if desired
+	if !noDepositFlag {
+		// prompt the user for a depositAmount
+		var toDeposit = big.NewInt(int64(depositAmountFlag))
+		if toDeposit.Cmp(big.NewInt(0)) == 0 {
```

Comment:
https://golang.org/pkg/math/big/

> The zero value for an Int, Rat, or Float correspond to 0. Thus, new values can be declared in the usual ways and denote 0 without further initialization:

I think we should use that instead of creating new zero value instances all the time

Reply:
cannot convert 0 (untyped int constant) to *big.Intcompiler

Reply:
Resoved. Thanks for the tip!